features/system_specs/system_specs.feature

```diff
@@ -19,7 +19,15 @@ Feature: System specs
 
   RSpec **does not** use your `ApplicationSystemTestCase` helper. Instead it
   uses the default `driven_by(:selenium)` from Rails. If you want to override
-  this behaviour you can call `driven_by` manually in a test.
+  this behaviour you can call `driven_by` manually in a test. Alternatively, 
+  if you want to specify a default driver for all system specs (rather than
+  using `driven_by` in every spec), add the following to `spec/rails_helper.rb`:
```

Comment:
```suggestion
  this behaviour you need to call `driven_by` in your specs.

  This can either be done manually in the spec files themselves or
  you can use the configuration helpers to do this for every system spec,
  for example by adding the following to `spec/rails_helper.rb`:
```